test/parallel/test-fs-read-stream-file-handle.js

```diff
@@ -80,3 +80,22 @@ fs.promises.open(file, 'r').then((handle) => {
     assert.strictEqual(output, input);
   }));
 }).then(common.mustCall());
+
+fs.promises.open(file, 'r').then((handle) => {
+  const controller = new AbortController();
+  const { signal } = controller;
+  const stream = handle.createReadStream({ signal });
```

Comment:
Nit: the test should also be expanded to verify that the `signal` is appropriately type checked.